crates/css-syntax-types/src/lib.rs

```diff
@@ -45,48 +43,92 @@ impl From<&Css> for Css {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-#[serde(deny_unknown_fields)]
```

Comment:
Why do we start allowing unknown fields?

Reply:
I do not see a problem with ignoring unknown fields. The previous version was not consistent on this, so I removed those restrictions.